README.md

```diff
@@ -1 +1,198 @@
+```
+
+### Configuring session to share file access
+
+By default on the USGS clusters, any time someone modifies a file, the file permissions change so that others are locked out. This made it really difficult to work collaboratively in one directory. To get around this, first give read-write access to everyone in your group for this whole directory (one time):
+```sh
+chmod g+w <filename or directory>
+```
+
+Then, for every session you start up, configure your session so that any changes you make to a file will not modify the file's permissions. Note that this is not something you want to do if you are working on files that only you should be able to change, e.g. your id_rsa keys.
+
+```sh
+umask 002
+```
+
+### Editing files on the cluster - Launching Jupyter Lab
+
+Similar instructions are available for [tallgrass - lake temps](https://github.com/USGS-CIDA/lake-temperature-neural-networks/tree/master/2_model#editing-files-on-the-cluster), [yeti generally](https://hpcportal.cr.usgs.gov/hpc-user-docs/Yeti/Guides_and_Tutorials/how-to/Launch_Jupyter_Notebook.html), and [yeti - lake temps](https://github.com/USGS-R/lake-temperature-out/blob/master/README.md#editing-files-on-the-cluster---launching-jupyter-lab).
+
+You can use `vim` to edit files locally.
+
+You can also use the Jupyter interface to edit files via a browser-based IDE.
+
+1. In a terminal window:
+```sh
+ssh denali.cr.usgs.gov
+cd /caldera/projects/usgs/water/iidd/data-sci/lake-temp/res-temperature-process-models
+sbatch launch-jlab.slurm
+cat tmp/jlab.out
+```
+
+From the top of the output of the `cat` call, locate the `ssh` command between lines of `####`. Copy this command and paste it into a new terminal window (which will become permanently tied up).
+
+From the last line of output of the `cat` call, copy the `http://127.0.0.1` URL and paste it into a local browser window. A jupyter lab session should open in your browser.
+
+#### Preparing a software environment to run Jupyter and project code (once per user)
+
+You may need to install miniconda3 in your home directory on caldera, or find some other way to access conda, before the following directions become possible.
+
+To create a Jupyter-ready conda environment:
+```sh
+conda create -n jlab jupyterlab -c conda-forge
+```
+
+To add an R kernel to the Jupyter Lab IDE (so that we can build and run R notebooks in addition to Python notebooks):
+```sh
+conda activate jlab
+conda install -c r r-irkernel zeromq
+```
+If you have already set up Jupyter Lab for the project (see below) and launched Jupyter Lab, you will have to re-launch Jupyter Lab (see above) to see the R kernel.
+
+A local Rlibs folder and an .Renviron file that points to it are included in the git repo for this project. If it's needed, installation of GitHub packages may go more smoothly if you run the installation from a login node (`ssh caldera-dtn.cr.usgs.gov`) rather than a Denali node.
+
+### Using the GLM container locally (with Docker)
```

Comment:
It may be worth clarifying again (like you mentioned above), that the plan is to use your GLM3r docker container on Denali for all substantive model runs, so installation/use of David's GLM container locally isn't necessary to work on this code/run the models.